annotation/core/templates/annotate.html

```diff
@@ -299,13 +307,67 @@ <h2 class="text-muted"> <strong>Is the following written by a person or by a mac
             </div>
 
             <div class="card-body" id="analysis" style="display: none">
-              <p class="text-muted">
-                Thank you for your response!
+              <p class="text-info">
+                Why do you think this sentence is computer-generated? Select all that applies. 
               </p>
               <form id="revision-form">
                 <div>
-                  <small class="text-muted">Why do you think this sentence is computer-generated?</small>
-                  <input type="text" id="revision" name="revision" style="margin-bottom: 1rem; margin-top: 1rem" class="form-control" placeholder="Describe the problem you noticed here..." />
+                  <div class="text-dark">Basic Fluency Errors: The sentence...</div>
+                  <div class="form-check">
```

Comment:
Let's not worry about this for now, but at a later point, it might make sense to add the HTML options to the form dynamically by reading from the database so that we guarantee the form options in the HTML always exactly correspond with what's in the database.

Reply:
+1

Reply:
changed it to dynamically generated now, but "fluency" and "substance" subdivision is hardcoded right now

Reply:
Looks good!